fiasko_bro/code_helpers.py

```diff
@@ -51,3 +51,5 @@ def is_repo_too_large(path_to_repo, max_py_files_count, path_to_original_repo=No
     if path_to_original_repo:
         num_of_py_files_original_repo = count_py_files(path_to_original_repo)
         return num_of_py_files_original_repo > max_py_files_count
+
```

Comment:
According to PEP8, you have to use blank lines sparsingly, so this blank line is really superfluous. But we'll merge anyway.